pq/falcon_certverify.c

```diff
@@ -25,14 +25,23 @@
 #include <wolfssl/wolfcrypt/error-crypt.h>
 #include <wolfssl/test.h>
 
-int main(void)
+int main(int argc, char **argv)
 {
     int ret;
     WOLFSSL_CERT_MANAGER* cm = NULL;
 
     const char* caCert     = "./falcon_level5_root_cert.pem";
     const char* verifyCert = "./falcon_level5_entity_cert.pem";
 
```

Comment:
I see them under the current directory instead of under ./certs. If this is intentional, I would leave them.

Reply:
Yes. This is correct as the README instructions tell the user to copy those files into this directory.